local/linux/README.md

```diff
@@ -47,9 +48,24 @@ To terminate the local instance run the below
   * `oc cluster down`
 
 To reset the environment to a clean instance of origin with ASB and Service Catalog run the below
-  * `cd local`
+  * `cd local/linux`
   * `./reset_environment.sh`
 
 ### Tested with
   * ansible 2.2.2.0 & 2.3.0.0
     * Problems were seen using ansible 2.0
+
+### Troubleshooting
+
+**pull() got an unexpected keyword argument 'decode'**
+
+```
+Error pulling image docker.io/ansibleplaybookbundle/ansible-service-broker-apb:summit - pull() got an unexpected keyword argument 'decode'
+```
+
+This is a problem with the version of docker-py. More info in https://github.com/ansible/ansible-modules-core/issues/5515.
+Update to version 1.10.6 of docker-py
+
+```
+<sudo> pip install --upgrade docker-py
```

Comment:
Let's remove the suggestion to upgrade docker-py, instead let's recommend to remove 'docker-py' in favor of 'docker' pip module.

We've seen other errors related to having docker-py being present:
 https://github.com/fusor/catasb/issues/52 

We used to enforce the upgrade to 1.10.6 of docker-py, then we ended up upgrading to 'docker'.  The install of 'docker' pip module is handled here:

https://github.com/fusor/catasb/blob/master/ansible/roles/openshift_setup/tasks/main.yml#L100